Mathlib/RingTheory/KrullDimension/Module.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2025 Nailin Guan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Nailin Guan
+-/
+import Mathlib.RingTheory.KrullDimension.NonZeroDivisors
+import Mathlib.RingTheory.Support
+
+/-!
+
+# Dimension of Module
+
+In this file we defined `dim(M)` for a `R`-module `M`.
+
+-/
+
+namespace Module
+
+open Order
+
+variable (R : Type*) [CommRing R] (M : Type*) [AddCommGroup M] [Module R M]
+
+/-- The dimension of module, defined as `krullDim` of its support. -/
```

Comment:
```suggestion
/-- The krull dimension of module, defined as `krullDim` of its support. -/
```
And there's some ambiguity here whether support is viewed as a set of prime ideals or a set of ideals. I wonder if there is a way to make this clearer.